playbooks/portals-docker-command.yml

```diff
@@ -0,0 +1,55 @@
+- name: Run Docker Command on Skynet Webportal
+  hosts: webportals
+  remote_user: "{{ webportal_user }}"
+  gather_facts: False
+
+  # Target 3 servers at a time.
+  serial: 3
+
+  # Stop on first error, do not execute on the next host
+  any_errors_fatal: True
+
+  # Playbook specific vars
+  vars:
+    max_hosts: "{{ groups['webportals'] | length - 1 }}"
+
+  tasks:
+    # Check '--limit' is used. This is to protect against taking the entire
+    # fleet down
+    - name: Fail if you are targeting all dev and prod webportals
+      include_tasks: tasks/host-limit-check.yml
```

Comment:
Do we want to have this check for this one?

Reply:
@ro-tex  yes, this check should always be in. This prevents someone from accidentally running a command against all possible servers, production, dev, staging, etc.